detection-rules/attachment_emotet_heavily_padded_doc_in_zip.yml

```diff
@@ -0,0 +1,32 @@
+name: "Attachment: Emotet heavily padded doc in zip file"
+description: |
+  Detects a potential Emotet delivery method using padded .doc files that compress into small zip files. 
+  Contents may include Red Dawn templates exceeding 500MB.
+references:
+  - "https://twitter.com/Cryptolaemus1/status/1633099154623803394"
+type: "rule"
+severity: "high"
+source: |
+  type.inbound
+  and any(attachments, 
+    .file_extension == "zip"
+    and any(file.explode(.), 
+      .depth == 0 and .size < 1000000
+      and not .depth > 0
```

Comment:
```suggestion
      .depth == 0 and .size < 1000000
```

Reply:
hmm. I want to ensure there's never any object at .depth >0

Reply:
it might be clearer as 2 calls to fileexplode, where we check the depth separately in each? since depth can never be both 0 and greater than 0 on the same file that's analyzed

but i'm not sure it actually matters anyway - as is should have the same effect i think? cc @rw-access

Reply:
`not .depth > 0` will automatically be rewirrten to `.depth <= 0`, so this rule will evaluate: `.depth == 0 and .size < 1000000 and .depth <= 0`.

Reply:
TBH, I'm pretty confused by the rule. Something like this makes more sense to me intuitvely, if I'm understanding right. WDYT @jkamdjou @morriscode?

```sql
  type.inbound
  and any(attachments, 
    .file_extension == "zip"
    and .size < 1000000
    and any(file.explode(.),
      and strings.ends_with(.scan.exiftool.zip_file_name, ".doc")
      and .scan.exiftool.zip_uncompressed_size > 500000000
    )
  )
  and (
    (
      sender.email.domain.root_domain in $free_email_providers
      and sender.email.email not in $sender_emails
    )
    or (
      sender.email.domain.root_domain not in $free_email_providers
      and sender.email.domain.domain not in $sender_domains
    )
  )
```